compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/platform/SkiaParagraph.skiko.kt

```diff
@@ -625,12 +630,16 @@ fun FontStyle.toSkFontStyle(): SkFontStyle {
 }
 
 // TODO: Remove from public
-fun TextDecoration.toSkDecorationStyle(color: Color): SkDecorationStyle {
+fun TextDecoration.toSkDecorationStyle(
+    color: Color,
+    textDecorationLineStyle: TextDecorationLineStyle?
```

Comment:
Adding parameters to existing public methods breaks binary compatibility.

Reply:
Not sure why this function is public (@igordmn ?). It also says that one line above in a comment.

I understand it's out policy that `androidx.compose` APIs are not supposed to expose anything `org.jetbrains`.

Reply:
Let's make it internal and keep old public overload with hidden deprecation to avoid breaking change

Reply:
I thought it wasn't resolved, but it looks like just API dump is not updated after this fix

Reply:
I'll re-update it

Reply:
Done.